insights/core/__init__.py

```diff
@@ -1663,6 +1663,65 @@ def path_entry(self, path):
         return self.listings[directory]['entries'][name]
 
 
+class DiskUsage(CommandParser, dict):
+    """
+    Reads output of du command and turns it into a dictionary with pathname as
+    key and size in blocks.
+
+    Supports parsing input data as long as output is 2 column data with first
+    column as space size with integer values only and second as pathname which
+    can be a file or directory. Space size with decimal values or unit suffixes
+    like M, GiB is not supported.
+
+    du command produces output in 1K blocks unless block size is specified in
+    command options or an environment variable. This parser is intended to be
+    used only with default block size of 1K which is also equal to plain "du"
+    or "du -k".
+
+    Sample input data:
+
+        56      /var/lib/alternatives
+        4       /var/lib/logrotate
+        5492	/var/lib/mlocate
+        20      /var/lib/NetworkManager
+        186484  /var/lib/pgsql
+        856	    /var/lib/rhsm
+        110712	/var/lib/rpm
+        4	    /var/lib/rsyslog
+        64	    /var/lib/systemd
+        15200	/var/lib/yum
```

Comment:
This looks like it might be a cut/paste formatting error with the alignment of the paths?

Reply:
There were mix of spaces and tabs before here. I have corrected them with only tab characters as in original output but it still shows misaligned in github. I do not know how single tab characters are adjusted in terminal to show like fix width column data. If same file is viewed in vim,  the columns are aligned.